### ISSUE TITLE
Consider renaming

### ISSUE THREAD

**Issue by js6pak** [IEvangelist/dotnet-github-actions-sdk#5]
Having a `GitHub.` prefix makes it look like it's an official GitHub project, which it's not.
The fact that the c# project name, nuget package id and github repository names are all different is quite bizzare aswell.
My vote goes for `ActionsToolkitSharp`.

**Comment by IEvangelist** [comment]
Yeah, I was back and forth a lot on the naming. The `GitHub.` prefix isn't reserved, so I went with that. What about `GitHubActions.` instead?

**Comment by js6pak** [comment]
`GitHubActions.` still sounds too generic, I feel like having a more unique name for the project would be both less confusing and better for searchability.

**Comment by IEvangelist** [comment]
> `GitHubActions.` still sounds too generic, I feel like having a more unique name for the project would be both less confusing and better for searchability.

I was referring specifically to the NuGet package and project names:

| Current name | Proposed name |
|--------------|---------------|
| `GitHub.Actions.Core` | `GitHubActions.Core` |
| `GitHub.Actions.Glob` | `GitHubActions.Glob` |
| `GitHub.Actions.Octokit` | `GitHubActions.Octokit` |

I'm not a fan of the suffix `*Sharp` in the name and `Toolkit` might make sense. Maybe something like this: `GitHubActions.Toolkit.Core`?

**Comment by js6pak** [comment]
At least for me, `GitHubActions.Toolkit.Core` seems to suggest that it's the `Core` subcomponent of the `Toolkit` library owned by the `GitHubActions` organization.

Something like `GitHubActionsToolkit.Core` where `Core` is the subcomponent of `GitHubActionsToolkit` library would work better. It's still a little generic but it should be fine.

**Comment by IEvangelist** [comment]
Yeah, I'm good with that. I'll make the name change soon. Thanks for the suggestion.
